deploy/helm/templates/beyla/configmap.yaml

```diff
@@ -5,30 +5,26 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "common.fullname" (tuple . "-beyla-config") }}
+  name: {{ include "common.fullname" (tuple . "-obi-config") }}
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "common.labels" . | indent 4 }}
     app.kubernetes.io/component: config
   annotations:
 {{ include "common.annotations" . | indent 4 }}
 data:
-  beyla-config.yml: |
+  obi-config.yml: |
     {{- if not .Values.network_topology.config.otel_metrics_export }}
     otel_metrics_export:
       endpoint: http://swo-gateway:{{ .Values.otel.gateway.otlp_endpoint.port }}
       protocol: grpc
       interval: {{ quote .Values.otel.metrics.prometheus.scrape_interval }}
-      features: ["application", "application_process", "application_service_graph", "network"]
+      features: ["application", "application_service_graph", "network"]
     {{- end }}
     {{- if not .Values.network_topology.config.ebpf }}
     ebpf:
       context_propagation: all
     {{- end }}
-    {{- if not .Values.network_topology.config.process }}
-    process:
-      enabled: true
```

Comment:
I could not find any information about this setting (including searching in the source code).